docs/60-advanced.md

```diff
@@ -42,6 +42,11 @@ __bricks/user-profile.html__
         fruits: [{ name: 'Apple', icon: '🍎' }, { name: 'Pineapple', icon: '🍍' }]
       }
     }
+        
+    edit() {
+       console.log("Edit my profile..")
```

Comment:
Good catch!
Just to be picky, can you make 1 final point only or did you mean `…`? (There are 2 right now)

Reply:
Hmm, yes. I did not implemented a full solution to the problem :)

Reply:
I guess we can already merge this as it and improve later 👍